src/ngraph/runtime/cpu/mkldnn_emitter.hpp

```diff
@@ -234,6 +256,25 @@ namespace ngraph
                             scale,
                             ops);
                     }
+                    else if (std::is_same<OP, ngraph::op::QuantizedConvolutionBias>())
+                    {
+                        // conv+bias = cvt_to_int8(scale*(dst + bias))
+                        const float scale =
+                            quantization_util::get_scale<ngraph::op::QuantizedConvolutionBias>(
```

Comment:
this computation should depend on the convolution output type.

Reply:
Why? I don't get your point here.